src/ucp/core/ucp_request.inl

```diff
@@ -176,12 +176,23 @@ UCS_PTR_MAP_IMPL(request, 0);
     }
 
 
+#define UCP_REQUEST_CHECK_OPT_FLAGS(_param) \
+    if (ENABLE_PARAMS_CHECK && \
+        ucs_test_all_flags((_param)->op_attr_mask, \
+                           (UCP_OP_ATTR_FLAG_FAST_CMPL | \
+                            UCP_OP_ATTR_FLAG_MULTI_SEND))) { \
+        ucs_error("invalid optimization flags combination: %u", \
```

Comment:
We should return `UCS_STATUS_PTR(UCS_ERR_INVALID_PARAM)` in case of error. (if this was a warning, we would continue).